pkg/restapi/operation/didexchange/didexchange.go

```diff
@@ -49,7 +50,7 @@ type provider interface {
 }
 
 // New returns new DID Exchange rest client protocol instance
-func New(ctx provider) (*Operation, error) {
+func New(ctx provider, webhookNotifier webhook.Notifier) (*Operation, error) {
```

Comment:
webhook.Notifier should be an interface (can inject different implementation).

Reply:
Done!

Reply:
Not sure if we need interface or struct here, I was expecting array of strings here in args.
Webhook instance should be private and initialized internally.
`didexchange.Operation` is not customizable, it just returns list of http handles.
(Remember, restapi component is out of `*Aries`)

Reply:
using interface is fine.
Will be helpful in future to switch between web socket & web hooks.

but rename var to `notifier`